jerry-core/ecma/operations/ecma-exceptions.h

```diff
@@ -59,6 +59,13 @@ ecma_value_t ecma_raise_syntax_error (const char *msg_p);
 ecma_value_t ecma_raise_type_error (const char *msg_p);
 ecma_value_t ecma_raise_uri_error (const char *msg_p);
 
+#define ECMA_DEFINE_ERROR_VALUE(var_name, error_type, error_msg_p) \
+  ecma_value_t var_name = ecma_make_simple_value (ECMA_SIMPLE_VALUE_EMPTY); \
+  do { \
+    ecma_object_t *error_obj_p = ecma_new_standard_error_with_message (error_type, error_msg_p); \
+    var_name = ecma_make_error_obj_value (error_obj_p); \
+  } while (0)
```

Comment:
Could you please remove this macro? It doesn't seem to do much anymore, outside of replacing one line for another.